tests/fixtures.py

```diff
@@ -115,6 +116,14 @@ def mscolab_session_server(mscolab_session_app, mscolab_session_managers):
     handles per-test cleanup as well.
     """
     with _running_eventlet_server(mscolab_session_app) as url:
+        # Wait until the Flask-SocketIO server is ready for connections
+        while True:
+            try:
+                with socketio.SimpleClient() as sio:
+                    sio.connect(url)
```

Comment:
does it make a difference with sio.connect(url, transports='polling')

Reply:
No, it doesn't. Somehow the tests that request this fixture are permanently stuck in this loop now, as if the server never gets ready.